.eslintrc.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
We need to update this, probably with one similar to mobile (and also update desktop too). https://github.com/ShabadOS/desktop/issues/487